themes/default/content/blog/pulumi-cli-time-elapsed/index.md

```diff
@@ -0,0 +1,185 @@
+---
+title: "Pulumi CLI now displays Time Elapsed per resource"
+date: 2022-11-02T11:45:35-06:00
+draft: false
+meta_desc: See deployment times across different types of AWS and GCP resources.
+authors:
+    - kyle-dixler
+tags: ["pulumi", "AWS", "GCP", "features"]
```

Comment:
I'd probably leave out AWS and GCP, only because this post doesn't really apply specifically to them.

```suggestion
tags: 
  - pulumi
  - features
```